### PR TITLE
Updating index_shadow.html.erb Translation

### LINKED ISSUE #9251: Questions: index_shadow.html.erb Translation
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the files below

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

https://github.com/publiclab/plots2/blob/452a75bbbd1bf9cf8f0ab545a8f6a4eee3ec01a8/app/views/questions/index_shadow.html.erb#L7

Make this change
```diff
-<h5>What's your question?</h5>
+<h5><%=translation('questions.index_shadow.what_is_your_question')%></h5>
```
https://github.com/publiclab/plots2/blob/452a75bbbd1bf9cf8f0ab545a8f6a4eee3ec01a8/app/views/questions/index_shadow.html.erb#L9
Make this change
```diff
-<p>To ask a question, please <a class="requireLogin" href="/questions">log in</a> or <a class="requireLogin" href="/questions">sign up</a> first.</p>
+<p><%=translation('questions.index_shadow.to_ask_a_question_please')%><a class="requireLogin" href="/questions"><%=translation('questions.index_shadow.log_in')%></a><%=translation('questions.index_shadow.or')%> <a class="requireLogin" href="/questions"><%=translation('questions.index_shadow.sign_up')%></a><%=translation('questions.index_shadow.first')%></p>

```
In the translations file en.yml, make these changes
https://github.com/publiclab/plots2/blob/452a75bbbd1bf9cf8f0ab545a8f6a4eee3ec01a8/config/locales/en.yml#L664

Inside the `questions:` block, beneath the last indented section add:
```diff
  index_shadow:
      what_is_your_question: "What's your question?"
      to_ask_a_question_please: "To ask a question, please"
      log_in: "log in"
      or: "or"
      sign_up: "sign_up"
      first: "first."
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by aohslund**
Fixes #9251

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -1211,3 +1211,10 @@ en:
       long: "%B %d, %Y %H:%M"
       short: "%d %b %H:%M"
     pm: pm
+  index_shadow:
+    what_is_your_question: "What's your question?"
+    to_ask_a_question_please: "To ask a question, please"
+    log_in: "log in"
+    or: "or"
+    sign_up: "sign_up"
+    first: "first."
```
Comment (RuthNjeri) [inline config/locales/en.yml:1220]: @aohslund thanks for working on this. There's a question block in line 664 
`index_shadow:` should be indented inside the question block
https://github.com/publiclab/plots2/blob/452a75bbbd1bf9cf8f0ab545a8f6a4eee3ec01a8/config/locales/en.yml#L664

**Review by RuthNjeri — CHANGES_REQUESTED** [review]

File: config/locales/en.yml (hunk 1 above)
Comment (aohslund) [reply config/locales/en.yml]: Pushed new commit with index_shadow: moved!

**Review by cesswairimu — COMMENTED** [review]
Looks great :tada:  thanks @aohslund for working on this

**Comment by cesswairimu** [comment]
@RuthNjeri kindly give this another review when you get a sec, so that we are able to merge. Thanks

**Comment by RuthNjeri** [comment]
Thanks @cesswairimu! It looks good to me!

**Review by RuthNjeri — APPROVED** [review]
Thanks @aohslund !

**Comment by cesswairimu** [comment]
Thanks all..

**Pull request merged**